Productionize 53

jwaltireland opened the pull request:
Changes to support deployment to prod

petersonjdNIH reviewed (commented):
@jwaltireland Please name the two environment deployment workflow files consistently.

jwaltireland commented:
> @jwaltireland Please name the two environment deployment workflow files consistently.

Deleted the deploy-to-test.yml, as it's not longer needed. the deploy-env.yml uses github environment/environment secrets.   Also renamed destroy.yml to destroy-env.yml.
